lib/network_utils.ml

```diff
@@ -670,6 +671,19 @@ module Fcoe = struct
                         []
 end
 
+module SRIOV = struct
+    let call ?(log=false) args =
+        call_script ~log_successful_output:log ~timeout:(Some 10.0) !sriov_plugin args
+                
+    let get_capabilities name =
+        try
+            let output = call ["--capable"; name] in
+            if String.trim output = "1" then ["sriov"] else []
+        with _ ->
+            error "Failed to get SRIOV support status on device %s" name;
```

Comment:
error -> debug

Reply:
Other things to put into the error message:

* the string of the exception
* the location of this: `__LOC__` - this makes debugging a lot easier